### PR TITLE
Added test for 'move-node' operation

### PR DESCRIPTION
**Pull request by anishagg17**
**Description**
Added test for the `move-node` operation when `path` is not equal to the `new path`.

**Checks**
- [x] The new code matches the existing patterns and styles.
- [x] The tests pass with `yarn test`.
- [x] The linter passes with `yarn lint`. (Fix errors with `yarn fix`.)
- [x] The relevant examples still work. (Run examples with `yarn start`.)
~- [ ] You've [added a changeset](https://github.com/atlassian/changesets/blob/master/docs/adding-a-changeset.md) if changing functionality. (Add one with `yarn changeset add`.)~

### CODE REVIEW

**Comment by BrentFarese** [comment]
@anishagg17 is this test case in response to an issue/bug or just something you noticed we were lacking in test coverage?

**Comment by anishagg17** [comment]
yes, @BrentFarese the codeBase was lacking in test coverage.

**Review by cboppert — APPROVED** [review]
Looks good to me, is there a slightly more complex case we want to test at all? Or some sort of failure case?

I'm not sure. Will leave up to @BrentFarese or @timbuckley for final approval (or another maintainer a bit more active)

**Review by timbuckley — APPROVED** [review]
Looks good to me

**Pull request merged**
